lib/msal-common/src/client/BaseClient.ts

```diff
@@ -65,28 +64,7 @@ export abstract class BaseClient {
         this.networkClient = this.config.networkInterface;
 
         // Default authority instance.
-        this.defaultAuthorityInstance = AuthorityFactory.createInstance(
-            this.config.authOptions.authority || Constants.DEFAULT_AUTHORITY,
-            this.networkClient
-        );
-    }
-
-    /**
-     * Create authority instance if not set already, resolve well-known-endpoint
-     * @param authorityString
-     */
-    protected async createAuthority(authorityString: string): Promise<Authority> {
```

Comment:
Is this not needed for browser?

Reply:
Yep it is. Originally this PR only contained changes for node. Just added the same for browser. This is now in PublicClientApplication in browser.